test/system/010-images.bats

```diff
@@ -312,4 +312,19 @@ Deleted: $pauseID"
     is "$output" ""
 }
 
+@test "podman image history" {
```

Comment:
There's already a `110-history.bats`, I wonder if some of this could be piggybacked there?